### ISSUE TITLE
Invalid pkgconfig/libpqxx.pc file after building and installing .deb

### ISSUE THREAD

**Issue by ineffective** [jtv/libpqxx#264]
Hi, I were in need of newer libpqxx on Debian 9.9, so I built static package for my own used. I failed to compile one of my projects when using it, and after looking around for some time I found that libpqxx.pc file contains the following:

```shell
prefix=XXXX
exec_prefix=XXXX/arm-linux-gnueabihf
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: libpqxx
Description: C++ client API for the PostgreSQL database management system.
Version: 6.4.4
Libs: -L${libdir} -L -lpqxx
Cflags: -I${includedir} -I/usr/include/postgresql
```
Note the `-L -lpqxx` fragment in the `Libs:` line. After replacing it with only `-lpqxx` everything compiles fine.

I used archive [6.4.5.tar.gz](https://github.com/jtv/libpqxx/archive/6.4.5.tar.gz). It is configured with the following:

```shell
CXX=arm-linux-gnueabihf-g++-9.2.0 CC=arm-linux-gnueabihf-gcc-9.2.0 ./configure --host=arm-linux-gnueabihf --with-postgres-include=/usr/include/postgres --prefix=XXXX --exec-prefix=XXXX/arm-linux-gnueabihf --disable-documentation
```
Of course XXXX is a real path on my system (it already exists and contains other custom packages).

**Comment by jtv** [comment]
This one sounds familiar...  Is it possible that it's already been fixed in the current head?

**Comment by ineffective** [comment]
I didn't see any issue regarding this, but it doesn't mean that there isn't one. I'll try to build tip of `master` and see if it changes anything.

**Comment by ineffective** [comment]
Yup, I can confirm that it is fixed in current master (1ef49d1d5c762b66c8f0fbe0c3dae7b750a41653). I assume that it was fixed in 4519227d9329c8bd6e08be8b5db87f47f49fd711?

**Comment by jtv** [comment]
I'm not sure if that was actually the fix for this.  I have a faint memory of there being some incredibly stupid issue with whitespace as well.  But that may have been something else.

Anyway, I'll release 7.0 RSN.
